src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidator.java

```diff
@@ -12,11 +12,17 @@ public class AddressDtoValidator {
     public Errors validate(String parentAddressField, AddressDto addressDto, Errors errors, String loggingContext) {
         validatePropertyNameNumber(parentAddressField, addressDto.getPropertyNameNumber(), errors, loggingContext);
         validateLine1(parentAddressField, addressDto.getLine1(), errors, loggingContext);
-        validateLine2(parentAddressField, addressDto.getLine2(), errors, loggingContext);
+        if(addressDto.getLine2() != null) {
```

Comment:
Prefer using the Objects isNotNull() method to do these checks. Just a bit more readable.

Reply:
Updated this to use nonNull method.